src/components/KSelect/KSelectItem.vue

```diff
@@ -13,6 +13,7 @@
         type="button"
         :value="item.value"
         @click="handleClick"
+        @keydown="onKeyPress"
```

Comment:
@aanchalm01 I think here we can have two events `@keydown.up.prevent="$emit('arrow-up')"` and `@keydown.down.prevent="$emit('arrow-down')"` - this way we don't even need the `onKeyPress` function

Docs: https://vuejs.org/guide/essentials/event-handling#key-modifiers

Reply:
updated.